chain/types/signature_cgo.go

```diff
@@ -44,7 +44,7 @@ func (s *Signature) Verify(addr address.Address, msg []byte) error {
 		var sig bls.Signature
 		copy(sig[:], s.Data)
 
-		if !bls.Verify(&sig, digests, pubkeys) {
+		if !bls.Verify(sig, digests, pubkeys) {
```

Comment:
required for compilation

Reply:
Hmm, can you please propose this as a distinct PR to `master` (and confirm deps in the process)

Reply:
Ah, you may have to run `make clean`, its likely you have the old header files and libs for the bls code, and for some reason our makefile doesnt detect that they need to be updated

Reply:
Confirmed this change doesn't work for me, I need the original.